### ISSUE TITLE
Ошибка в получении полей формы публикации

### ISSUE THREAD

**Issue by Preci1** [plechev-64/wp-recall-current#8]
https://github.com/plechev-64/wp-recall-current/blob/ab54b148b715f0f35ef9a1c55e30d14b3c25dfde/classes/class-rcl-custom-fields.php#L726-L745

Для стандартной формы публикации типа post - все работает. Но для кастомных типов записей - нет, 

Проблема в строчке
`'rcl_fields_' . $post_type`
должно формироваться так
`'rcl_fields_' . $post_type.'_'.$id_form`

Но мне кажется там вообще switch не нужен т.к. ключ формируется как rcl_fields_{POST_TYPE}_{FORM_ID}

Надо переписать эту функцию

**Comment by Preci1** [comment]
`function rcl_get_custom_fields( $post_id, $post_type = false, $id_form = false ) {
	
	if(!$post_type) {
		$post_type = get_post_field('post_type', $post_id);
	}
	
	if(!$id_form) {
		$id_form = get_post_meta( $post->ID, 'publicform-id', 1 ) ?: 1;
	}
	
	$id_field	 = "rcl_fields_{$post_type}_{$id_form}";
	
	return apply_filters( 'rcl_custom_fields_post', get_site_option( $id_field ), $post_id, $post_type );
}`

Например так, если я правильно все понял.

Единственное, этот код не будет работать если поля формы публикации были сформированы в старой версии recall и после обновления не обновлялись - тогда для кастомных записей они будут лежать по адресу rcl_fields_{$post_type} - не знаю надо ли это учитывать, или пусть все пересохранят настройки форм публикаций

**Comment by plechev-64** [comment]
Верно, данная функция работает неверно, проблема данной функции в том, что она получает не только произвольные поля, но и стандартные и в текущей версии она, по сути, deprecated, но я похоже забыл написать ей замену.
В ближайшем обновлении сделаю, спасибо.
